docs/docs/development/local-toolchain/setup/container.mdx

```diff
@@ -0,0 +1,273 @@
+    operating system and selecting `Remote: Show Remote Menu`:
+
+    - **Windows/Linux**: `Ctrl + Shift + P`
+    - **MacOs**: `Cmd + Shift + P`
+
+    The first time you do this on your machine, it will pull down the Docker
+    image from the registry and build the container. **Subsequent launches are
+    much faster!**
+
+  </TabItem>
+  <TabItem value="cli">
+    First, make sure that the
+    [Dev Container CLI](https://github.com/devcontainers/cli) is installed by
+    running:
+
+    ```sh
+    devcontainer --version
+    ```
+
+    To be able to start the [Dev Container](https://containers.dev/), the
+    [Dev Container CLI](https://github.com/devcontainers/cli) has to know where
+    the `devcontainer.json` is located. This can be done using the
+    `--workspace-folder` option:
+
+    ```sh
+    devcontainer up --workspace-folder "/absolute/path/to/zmk"
+    ```
+
+    The first time you do this on your machine, it will pull down the Docker
+    image from the registry and build the container. **Subsequent launches are
+    much faster!**
+
+    Once the container is running, you can connect to it by using its container
+    ID. This allows you to execute commands inside the container:
+
+    ```sh
+    docker ps                                                   # List containers
+    docker exec -w /workspaces/zmk -it <container_id> /bin/bash # Connect
+    ```
+
+  </TabItem>
+  <TabItem value="podman">
+    First, make sure that Podman is installed by running:
+
+    ```sh
+    podman --version
+    ```
+
+    Since Podman is largely compatible with Docker, it is possible to use the
+    Dockerfile provided for use with Dev Containers for building a container
+    image.
+
+    ```sh
+    podman build -t <container-name> -f Dockerfile </path/to/.devcontainer>
+    ```
+
+    Once this is done, the container can be started with the `podman run`
+    command. Depending on which volumes you are using, you might have to remove
+    mounting options.
+
+    ```sh
+    podman run -it --rm \
+      --security-opt label=disable \
+      --workdir /workspaces/zmk \
+      -v /path/to/zmk:/workspaces/zmk \
+      -v /path/to/zmk-config:/workspaces/zmk-config \   # Removeable
+      -v /path/to/zmk-modules:/workspaces/zmk-modules \ # Removeable
+      -p 3000:3000 \
+      <container-name> /bin/bash
+    ```
+
+  </TabItem>
+</Tabs>
+
+## Configure Zephyr Workspace
+
+:::caution
+The following step and any future [build commands](../build-flash.mdx) must be
+executed from the command line _inside_ the container.
+:::
+
+```sh
+west init -l app/ # Initialization
+west update       # Update modules
+```
+
+:::tip
+This step pulls down quite a bit of tooling, be patient!
+:::
+
+:::note
+If you are using a Docker-based approach, you have to restart the container at
+this point. Stopping it is possible with these commands.
+
+```sh
+docker ps                    # List containers
+docker stop "<container-id>" # Stop the container
+```
+
+:::
```

Comment:
````suggestion
If you are using a Docker-based approach, you have to restart the container at
this point. Stopping it is possible with these commands.

```sh
docker ps                    # List containers
docker stop "<container-id>" # Stop the container
```
````
Box fatigue.